### PR TITLE
DOC-2851 Android updates can take an hour

### PR DESCRIPTION
**Pull request by catong**
Update mobile best practices doc with reminder that it can take up to an hour for newly published course changes to be updated on the Android app.
DOC-2851/MA-2259

Reviewers:
@BenjiLee 
@lamagnifica or @srpearce or @pdesjardins 
FYI @jaakana

### CODE REVIEW

**Comment by BenjiLee** [comment]
👍  For course update on android section

File: en_us/shared/reaching_learners/design_for_mobile.rst [hunk 1]
```diff
@@ -16,6 +16,10 @@ To make the course experience for mobile learners as rewarding as it is for
 learners using desktop computers, keep the following best practices in mind as
 you design, test, and run your course.
 
+* Course updates that you make might not be immediately reflected in the edX
```
Comment (lamagnifica) [inline en_us/shared/reaching_learners/design_for_mobile.rst]: I'm wondering if the comparison to the web browser might be worth making more explicit. ..
might not be immediately reflected in --> might be available sooner in a web browser than in (?)

**Comment by lamagnifica** [comment]
:+1: 
surfaced one possible rewording
Is there any other place that this same note might be useful? For example, in developing_course/course_components.rst, in the section on editing a component, or in the workflow.rst in that same directory, with the topic on revising?

**Comment by catong** [comment]
Travis test is failing due to a known issue totally unrelated to this PR: warning about a new anchor in the Research Guide, which won't be resolved until the target guide is rebuilt. 
Merging https://github.com/edx/edx-documentation/pull/994

**Pull request merged**